packages/block-editor/src/components/block-list/breadcrumb.native.js

```diff
@@ -19,52 +19,60 @@ import SubdirectorSVG from './subdirectory-icon';
 
 import styles from './breadcrumb.scss';
 
+const renderIcon = ( icon, key ) => {
+	if ( typeof icon.src === 'function' ) {
```

Comment:
Looking at the source of the Icon component, it should already do all of this by itself when you pass a function, not sure why this is needed:

https://github.com/WordPress/gutenberg/blob/164d5830a9acd895dfd661b5fde1ca8b80b270ac/packages/components/src/icon/index.js#L39-L49

Reply:
Let me check that because since I add this some time passed and maybe it is not necessary at this moment as you mention Icon should do that.

Reply:
confirmed, it should work with current master codebase. Will revert changes in next push